README.md

```diff
@@ -1 +1,52 @@
-# Hello-Microverse_
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
+This project is configuration of linter into html and CSS.
+
+## Built With
+
+- Major languages
+HTML and CSS
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] **GETTING STARTED** is used to give instructions on setting up your project locally. For example, they give you an instruction: **Modify this file to match your project, and remove sections that don't apply.** 
- [ ] "_To get a local copy up and running follow these simple example steps._" is just an example and we can delete it when replacing it with our information.
- [ ] Please 🙏🏽 use this section in this way. For example:
  - Clone this [project](git@github.com:AbdifatahYasin1/Hello-Microverse_.git)
  - Open it in your browser.